app/controllers/releases/lts.js

```diff
@@ -2,6 +2,10 @@ import Controller from '@ember/controller';
 
 export default class ReleasesLtsController extends Controller {
   currentlySupportedLTS = [
+    {
+      version: '3.20',
+      promotionDate: new Date('August 24, 2020')
```

Comment:
Ember `v3.20` was promoted to LTS when `v3.21.0` was released on August 24, 2020.

https://github.com/emberjs/ember.js/releases/tag/v3.21.0